File: JAVA/28. Find the Index of the First Occurrence in a String.java
class Solution {
    public int strStr(String h, String n) {
        if(h.length()<n.length())
        return -1;
        else
        {
            for(int i=0;i<h.length()-n.length()+1;i++){
            if(h.substring(i,i+n.length()).equals(n))
            {
                return i;
            }
        }}
        return -1;
    }
}

File: C++/28. Find the Index of the First Occurrence in a String.cpp
class Solution {
public:
    int strStr(string h, string n) {
        if(h.size()<n.size())
        return -1;
        else{
        for(int i=0;i<h.size()-n.size()+1;i++)
        {
            if(h.substr(i,n.size())==n)
            {
                return i;
                break;
            }
        }}
        return -1;
    }
};